change/@ni-nimble-components-63ba584c-0659-44df-8b5c-65ebf12532b9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If anchor-tabs are trying to behave like tabs, just with each tab having its own URL, it doesn't seem like it would ever be within the usage of the anchor-tab to open in a new tab/window by default. However, I think the same argument could probably be made for the anchor-tree-item as well.

Is your position that we should include the docs on all our anchor elements? I don't mind doing that. It would certainly be less presumptive about how the components are being used.

Reply:
Yea I think we can start by having the same guidance on all anchors. I don't think the advice is wrong in those cases, and if we refine it more we can specialize (i.e. anchor-tabs should likely never need a blank target, etc). Think it's better than not providing guidance in this case.